pmacApp/src/pmacController.h

```diff
@@ -524,7 +547,7 @@ class pmacController
     bool csResetAllDemands;
     int csCount;
     int Sys_CPUFreq_;
-    int Sys_CPUType_;
+    // int Sys_CPUType_;
     int Sys_BgSleepTime_;
```

Comment:
You're right James, Sys_BgSleepTime_ is not used anymore.

The following aren't being used as well:
  Sys_ServoPeriod_
  Sys_RtIntPeriod_
  Sys_PhaseOverServoPeriod_

Any thoughts about deleting them?

Reply:
I think any variable that is not being used should be deleted, otherwise we build up lots of commented out lines which can be confusing when another developer looks at the code at a later date.

Also, the change tracking from git means that we always have a record of what was there if we ever need to check anything later